lldb/tools/lldb-dap/src-ts/debug-adapter-factory.ts

```diff
@@ -0,0 +1,21 @@
+import * as vscode from 'vscode';

```

Comment:
Wanted to mention again that maybe we make a directory that contains the needed typescript sources that will be in the exact format needed for installing an extension? So if we have `lldb/tools/lldb-dap/dap-extension` which contains all of the sources in the exact hierarchy needed so to install the extension we just basically can do something like:
```
$ rsync -av lldb/tools/lldb-dap/dap-extension` ~/.vscode/extensions/llvm-org.lldb-dap-0.1.0
```
This would copy the entire directory hierarchy over so this directory would always be exactly as it will appear in the ~/.vscode/extensions directory.

Reply:
This approach doesn't work anymore. In fact, for the entirety of 2023 this was not possible.

Reply:
Perhaps your docker config is allowing you to do that, but you can see this old gh issue that talks more about this problem https://github.com/llvm/llvm-project/issues/63655#issuecomment-1746745547